examples/1_Santanilla_Nanomole_HTE/example_Santanilla.ipynb

```diff
@@ -0,0 +1,8234 @@
+       "43             C1(C2=CC=CC=C2)=CN=C(NC3=CN=CC=C3)C=C1  \n",
+       "44                      N=C(NC1=CN=CC=C1)CC2=CC=CC=C2  \n",
+       "45                         O=C(NC1=CN=CC=C1)OC(C)(C)C  \n",
+       "46                     C12=CC=CC=C1N(C3=CN=CC=C3)N=C2  \n",
+       "47                       C1(CCCOC2=CN=CC=C2)=CC=CC=C1  \n",
+       "48                          C1(OC2=CN=CC=C2)=CC=CC=C1  \n",
+       "49                          C1(SC2=CN=CC=C2)=CC=CC=C1  \n",
+       "50             C1(P(C2=CN=CC=C2)C3=CC=CC=C3)=CC=CC=C1  \n",
+       "51               C1(CN2C=C(C3=CN=CC=C3)C=N2)=CC=CC=C1  \n",
+       "52                     O=C(C(C1=CN=CC=C1)C(OCC)=O)OCC  \n",
+       "53              O=C(OCC)/C=C(C1=CC=CC=C1)/C2=CC=CN=C2  \n",
+       "54                    CCCCC1=CC=C(C#CC2=CN=CC=C2)C=C1  \n",
+       "55                                            CS(C)=O  \n",
+       "56                                            CC(O)=O  \n",
+       "57                           C1(C2=CC=CC=C2)=CC=CC=C1  "
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Load SMILES info from Chemdraw .csv\n",
+    "smiles_data = pd.read_csv('experiment_2_molecules.csv')\n",
+    "smiles_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Handle dative bonds in metal complexes\n",
+    "From Greg Landrum at RDKit: https://www.rdkit.org/docs/Cookbook.html#organometallics-with-dative-bonds<br>\n",
+    "Added phosphorous to dative atom list in set_dative_bonds and added new function has_transition_metal"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def is_transition_metal(atom):\n",
+    "    \"\"\"\n",
+    "    Determines if an atom is a transition metal\n",
+    "    \n",
+    "    ARGUMENTS:\n",
+    "        (rdkit.Chem.rdchem.Atom)atom: atom object for the atom in question\n",
+    "        \n",
+    "    RETURNS:\n",
+    "        (bool): boolean describing whether atomic number of atom is in \n",
+    "                one of transition-metal ranges\n",
+    "    \"\"\"\n",
+    "    n = atom.GetAtomicNum()\n",
+    "    return (n>=22 and n<=29) or (n>=40 and n<=47) or (n>=72 and n<=79)\n",
+    "\n",
+    "def has_transition_metal(mol):\n",
+    "    \"\"\"\n",
+    "    Determines if a molecule has a transition metal\n",
+    "    \n",
+    "    ARGUMENTS:\n",
+    "        (rdkit.Chem.rdchem.Mol)mol: mol object for the molecule in question\n",
+    "        \n",
+    "    RETURNS:\n",
+    "        (bool): boolean describing whether molecule has an atom with \n",
+    "                atomic number in the transition-metal ranges\n",
+    "    \"\"\"\n",
+    "    for atom in mol.GetAtoms():\n",
+    "        n = atom.GetAtomicNum()\n",
+    "        if (n>=22 and n<=29) or (n>=40 and n<=47) or (n>=72 and n<=79):\n",
```

Comment:
Call `is_transition_metal` here instead of duplicating code.

Reply:
Done. Good catch, thanks!